pkg/server/tokenrequest/tokenrequest.go

```diff
@@ -233,10 +250,16 @@ var tokenTemplate = template.Must(template.New("tokenTemplate").Parse(
   <code>{{.AccessToken}}</code>
 
   <h2>Log in with this token</h2>
-  <pre>oc login <span class="nowrap">--token={{.AccessToken}}</span> <span class="nowrap">--server={{.PublicMasterURL}}</span></pre>
+  <pre>
+    {{range $p := .LoginParts}}<span class="nowrap">{{$p}}</span>{{end}}
```

Comment:
Will this break escaping of HTML since it's now a `JSStr`?

I was thinking of just having `AccessToken` and `AccessTokenJSStr` in the template data and using the first in the HTML and the second in the JS copy clipboard code. FWIW the duplication doesn't bother me since one command has HTML markup and the other doesn't. They're not exactly the same. It's almost better to be explicit.

Reply:
> Will this break escaping of HTML since it's now a `JSStr`?

Hm, you are right, the parser will detect `<span class="nowrap">{{ something }}</span>` and will try to HTML escape `something` automatically which is probably not what we want.

OK, I will add simple strings instead of using "parts" as suggested.